tests/manage/test_verify_all_fields_in_sc_yaml_with_oc_describe_sc.py

```diff
@@ -0,0 +1,131 @@
+import pytest
+import logging
+
+from tests import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.ocs import OCS
+from ocs_ci.utility import templating
+from ocs_ci.framework.testlib import ManageTest, tier1
+from tests.fixtures import create_ceph_block_pool
+
+log = logging.getLogger(__name__)
+RBD_SC_OBJ = None
+CEPHFS_SC_OBJ = None
+
+
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    This is a test fixture
+    """
+    def finalizer():
+        teardown()
+    request.addfinalizer(finalizer)
+    setup()
+
+
+def setup():
+    """
+    Setting up the environment
+    *. Creating RBD and CephFS Secret
+    *. Creating Ceph block pool
+    """
+    global RBD_SECRET_OBJ
```

Comment:
Are `RBD_SECRET_OBJ` and `CEPHFS_SECRET_OBJ` used somewhere in the test code? If not then it would be better to work with them just in fixture and don't make them global variables (e.g. use them in one fixture function with `yield`).

Reply:
Done.
================================================================================== PASSES ===================================================================================
========================================================================== short test summary info ==========================================================================
PASSED test_verify_all_fields_in_sc_yaml_with_oc_describe_sc.py::TestVerifyAllFieldsInScYamlWithOcDescribe::test_verify_all_fields_in_sc_yaml_with_oc_describe[RBD]
PASSED test_verify_all_fields_in_sc_yaml_with_oc_describe_sc.py::TestVerifyAllFieldsInScYamlWithOcDescribe::test_verify_all_fields_in_sc_yaml_with_oc_describe[CEPHFS]
======================================================================== 2 passed in 240.22 seconds =========================================================================